src/node/__tests__/http-request.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpRequestImpl } from "../http-request"
+
+describe("HttpRequestImpl", () => {
+  it("should remove headers", () => {
+    const body = '"{param: 1}"'
+    const url = ""
+    const headers = {
+      someHeader: 'Some content',
+      someOtherHeader: 'Some other header content'
+    }
+    const method = "GET"
+    const request = new HttpRequestImpl(headers, body, method, url);
+    const expectedRequest = new HttpRequestImpl({ someOtherHeader: 'Some other header content' }, body, method, url);
+    expect(request.removeHeader("someHeader")).toEqual(expectedRequest);
+  })
+
+  it("should replace headers", () => {
+    const body = '"{param: 1}"'
+    const url = ""
+    const headers = {
+      someHeader: 'Some content',
+      someOtherHeader: 'Some other header content'
```

Comment:
Double quotes?

Reply:
I think we need to add .prettier for this

Reply:
Or I will run lint-fix script on every commit